eq-author/src/components/Comments/Comment/index.js

```diff
@@ -0,0 +1,225 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import CommentEditor from "../CommentEditor";
+import Collapsible from "components/Collapsible";
+import Button from "components/buttons/Button";
+
+import PureComment from "../PureComment";
+
+const StyledCollapsible = styled(Collapsible)`
+  .collapsible-title,
+  .collapsible-title > * {
+    font-weight: normal;
+    padding: 0;
+  }
+
+  .collapsible-header {
+    margin-bottom: 1em;
+  }
+
+  .collapsible-body {
+    border-left: none;
+    margin-top: 0;
+  }
+
+  ul {
+    padding: 0;
+
+    li {
+      display: block;
+    }
+  }
+`;
+
+const Replies = ({
+  replies,
+  commentId,
+  onUpdateReply,
+  onDeleteReply,
+  showAddReply,
+  showReplyBtn,
+}) => {
+  const numOfReplies = replies.length;
+
+  return (
+    <>
+      {numOfReplies === 0 && <React.Fragment />}
+      {numOfReplies > 0 && (
+        <StyledCollapsible
+          title={`${numOfReplies} ${numOfReplies > 1 ? "replies" : "reply"}`}
+          showHide
+          withoutHideThis
+          key={`replies-to-${commentId}`}
+        >
+          <ul>
+            {replies.map(({ id, ...rest }) => (
+              <li key={`list-item-comment-${id}`}>
```

Comment:
is the `<li>` outside the `<Comment>` because it might be a single entity at some point?

Reply:
Yeah.